app/validators/class_name_validator.rb

```diff
@@ -0,0 +1,10 @@
+class ClassNameValidator < ActiveModel::EachValidator
+  def validate_each(record, attr_name, value)
+    return false if value.nil?
+    begin
+      Object.class_eval(value)
```

Comment:
I thought about updating this to be more specific to model names, but the problem is that `ApplicationRecord.descendants` only returns a list of classes that have already been loaded, which would mean that the validator would need to eager load the rest of the models in non-production environments in order for it to work properly. This seems to be a simpler way to ensure that at least what's provided is indeed a class inside the application, even though it's less specific.